### PR TITLE
feat: Add track to trajectory converter

### PR DESCRIPTION
**Pull request by paulgessinger**
Add a dedicated conversion algorithm from tracks to trajectories.

Blocked by:
- #1839

### CODE REVIEW

File: Examples/Algorithms/Utilities/include/ActsExamples/Utilities/TracksToTrajectories.hpp [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
```
Comment (andiwand) [inline Examples/Algorithms/Utilities/include/ActsExamples/Utilities/TracksToTrajectories.hpp:3]:
```suggestion
// Copyright (C) 2023 CERN for the benefit of the Acts project
```

File: Examples/Algorithms/Utilities/src/TracksToTrajectories.cpp [hunk 2]
```diff
@@ -0,0 +1,89 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
```
Comment (andiwand) [inline Examples/Algorithms/Utilities/src/TracksToTrajectories.cpp:3]:
```suggestion
// Copyright (C) 2023 CERN for the benefit of the Acts project
```

**Review by andiwand — APPROVED** [review]
looks good :+1: 

this is temporary right? so I guess we can forget about doc

**Comment by paulgessinger** [comment]
I suppose so @andiwand, but we also don't have a strong documentation policy for the examples components, as far as I know.

**Review by andiwand — APPROVED** [review]

**Pull request merged**